common/dal2/src/dal.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::marker::PhantomData;
+use std::sync::Arc;
+
+use common_exception::Result;
+
+use crate::ops::*;
```

Comment:
IMHO we should avoid using `use xxx:*;`:
`use`  a trait may alter some behaviors.
And it might potentially introduce unexpected naming conflict.

Reply:
I will change it.